src/Polly.Core/Simmy/Outcomes/OutcomeChaosPipelineBuilderExtensions.TResult.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Polly.Simmy.Outcomes;
+
+namespace Polly.Simmy;
+
+/// <summary>
+/// Extension methods for adding outcome to a <see cref="ResiliencePipelineBuilder"/>.
+/// </summary>
+public static partial class OutcomeChaosPipelineBuilderExtensions
+{
+    /// <summary>
+    /// Adds a fault chaos strategy to the builder.
+    /// </summary>
+    /// <typeparam name="TResult">The type of result the retry strategy handles.</typeparam>
+    /// <param name="builder">The builder instance.</param>
+    /// <param name="enabled">A value that indicates whether or not the chaos strategy is enabled for a given execution.</param>
+    /// <param name="injectionRate">The injection rate for a given execution, which the value should be between [0, 1].</param>
```

Comment:
I'll stop commenting on these now, but we should apply the comment I've made previously to everywhere in the code that `[0, 1]` is present.